types/git-username/git-username-tests.ts

```diff
@@ -2,4 +2,5 @@ import username = require('git-username');
 
 username('some/path'); // $ExpectType string | null
 username({ cwd: 'some/path' }); // $ExpectType string | null
-username({ strict: true }); // $ExpectType string
+username({ strict: true }); // $ExpectType string | null
+username('some-path', { strict: true }); // $ExpectType string | null
```

Comment:
Similarly to the changes in the `index.ts`, the tests should look something like this:
```ts
import username = require('git-username');

// test type exports
type BasicOptions = username.BasicOptions;
type FullOptions = username.FullOptions;

username(); // $ExpectType string | null
username('some/path'); // $ExpectType string | null
username({ cwd: 'some/path' }); // $ExpectType string | null
username({ strict: true }); // $ExpectType string | null
username('some-path', { strict: true }); // $ExpectType string | null
username('some-path', { cwd: 'some/path' }); // $ExpectError
```